elevate-user/constants/routes.js

```diff
@@ -830,6 +834,8 @@ module.exports = {
 			orchestrated: false,
 			targetRoute: {
 				path: '/user/v1/account/validatingEmailIds',
+                type: 'POST',
+				functionName:'listUserDetailsByEmail'
```

Comment:
Why this widely different name? And why is the API called validatingEmailIds instead of just validateEmailIds?